Hadrons/Modules/MSource/Gauss.hpp

```diff
@@ -0,0 +1,120 @@
+#ifndef Hadrons_MSource_Gauss_hpp_
+#define Hadrons_MSource_Gauss_hpp_
+
+#include <Hadrons/Global.hpp>
+#include <Hadrons/Module.hpp>
+#include <Hadrons/ModuleFactory.hpp>
+
+BEGIN_HADRONS_NAMESPACE
+
+/******************************************************************************
+ *                         Gauss                                              *
+ * result[n] = 1/(sqrt(2*pi)*width)^dim                                       *
+ *            * exp(-|n-position|^2/(2*width^2))                              *
+ * where:                                                                     *
+ *   n=(n[0],n[1],...,n[dim-1])  (lattice coordinate)                         *
+ *   dim=Nd-1                                                                 *
+ ******************************************************************************/
+BEGIN_MODULE_NAMESPACE(MSource)
+
+class GaussPar: Serializable
+{
+public:
+    GRID_SERIALIZABLE_CLASS_MEMBERS(GaussPar,
+                                    std::string, position,
+                                    double,      width);
+};
+
+template <typename FImpl>
+class TGauss: public Module<GaussPar>
+{
+    BASIC_TYPE_ALIASES(FImpl,);
+public:
+    // constructor
+    TGauss(const std::string name);
+    // destructor
+    virtual ~TGauss(void) {};
+    // dependency relation
+    virtual std::vector<std::string> getInput(void);
+    virtual std::vector<std::string> getOutput(void);
+    // setup
+    virtual void setup(void);
+    // execution
+    virtual void execute(void);
+private:
+    std::vector<int> position_;
+};
+
+MODULE_REGISTER_TMP(Gauss, TGauss<FIMPL>, MSource);
```

Comment:
Could you please add a scalar version creating a complex field, cf. how it is handled in point sources

Reply:
The Gauss module actually does create a complex field. I can substitute FIMPL by ScalarImplCR, so that it does not unnecessarily load the fermion implementation. Is that what you mean?

Reply:
We want both versions, once more please see the point source case and do the same here. The scalar version can just be called `ScalarGauss` (cf. `ScalarPoint`).